pkg/scheduler/backend/queue/backoff_queue.go

```diff
@@ -111,6 +190,14 @@ func (bq *backoffQueue) isPodBackingoff(podInfo *framework.QueuedPodInfo) bool {
 	return boTime.After(bq.clock.Now())
 }
 
+// podCompletedBackoff returns true if a pod completed its backoff
+// and should be flushed to the activeQ. This method is different than isPodBackingoff,
```

Comment:
Can't we just change `isPodBackingoff` like this?
```
func (bq *backoffQueue) isPodBackingoff(podInfo *framework.QueuedPodInfo) bool {
	boTime := bq.getBackoffTime(podInfo)
	return boTime.After(bq.alignToWindow(bq.clock.Now()))
}
```

It's weird to have both `podCompletedBackoff` and `isPodBackingoff`. They should just be the opposite; if the backoffQ regards the pod has completed backoff (`podCompletedBackoff() == true`), the pod shouldn't be backing off (`isPodBackingoff() == false`), and vise versa.